### PR TITLE
docs(readme): slight improvements

### PR DESCRIPTION
**Pull request by Apercu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
 
-## Using with Browserify, Webpack, and other JavaScript Bundlers
+    npm install --save react-map-gl
 
 * `browserify` - react-map-gl is extensively tested with `browserify` and works without configuration.
 
 * `webpack 2` - Most of the provided react-map-gl examples use webpack 2. For a minimal example, look at the [exhibit-webpack](https://github.com/uber/react-map-gl/tree/master/examples/exhibit-webpack) folder, demonstrating a working demo using `webpack 2`.
 
-* `create-react-app` - At this point configuration-free builds are not possible with webpack due to the way the mapbox-gl-js module is published. You will need to eject your app (sorry) and add one line alias to your webpack config.
+* `create-react-app` - At this point configuration-free builds are not possible with webpack due to the way the mapbox-gl-js module is published. You will need to eject your app the alias to your webpack config, as showed in the exhibit-webpack.
```
Comment (abmai) [inline README.md]: Change last sentence to `You will need to add an alias to your webpack config, as shown in the exhibit-webpack`.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,38 @@
...
 
-# Installation
+<h5 align="center">
+React Components Suite for <a href="https://github.com/mapbox/mapbox-gl-js">Mapbox GL JS</a>.
+</h5>
 
-```sh
-npm install --save react-map-gl
-```
+### Introduction
```
Comment (abmai) [inline README.md]: Keep the old `introduction` text for now.

File: README.md [hunk 3]
```diff
@@ -35,7 +47,7 @@ class Map extends Component {
         latitude={37.7577}
         longitude={-122.4376}
         zoom={8}
-        onViewportChange={(viewport) => {
+        onViewportChange={viewport => {
```
Comment (abmai) [inline README.md]: Put the parens back.

Comment (Apercu) [inline README.md]: whyyyy

File: README.md [hunk 4]
```diff
@@ -45,25 +55,22 @@ class Map extends Component {
 }
 ```
 
-# About Mapbox Tokens
+### About Mapbox Tokens
+
+To show maps from a service such as Mapbox you will need to register on their website in order to retrieve an access token required by the map component, which will be used to identify you and start serving up map tiles. The service will be free until a certain level of traffic is exceeded.
 
-To show maps from a service such as Mapbox you will need to register with Mapbox and get a "token" that you need to provide to your map component. The map component will use the token to identify itself to the mapbox service which then will start serving up map tiles. The token will usually be free until a certain level of traffic is exceeded.
+There are several ways to provide a token to your app, and is showcased in some of the examples folder:
```
Comment (abmai) [inline README.md]: `and is` => `and are`

File: README.md [hunk 5]
```diff
@@ -45,25 +55,22 @@ class Map extends Component {
...
-* Set an environment variable (MapboxAccessToken) - through the use of a webpack loader or browserify transform, see the hello-world examples for details.
-* Provide a token in the URL.
+* Modify the source directly
+* Set the `MapboxAccessToken` environment variable
+* Provide it in the URL, e.g `?access_token=TOKEN`
 
-To make the maps load, either:
-* add `?access_token=TOKEN` to the URL where `TOKEN` is a valid Mapbox access token, or
-* set the `MapboxAccessToken` environment variable before running `npm start`
+But we would advice you to use something like [dotenv](https://github.com/motdotla/dotenv) and put your key in an untracked `.env` file, that will then expose it as a `process.env` variable, with much less leaking risks.
```
Comment (abmai) [inline README.md]: `But we would advice you to use something like` => `But we recommend using`

File: README.md [hunk 6]
```diff
@@ -45,25 +55,22 @@ class Map extends Component {
...
-## Using with Redux
+### Redux Usage
 
-If you're using redux, it is very easy to hook this component up to
-store state in the redux state tree. The simplest way is to take all
-properties passed to the `onChangeViewport` function property and add them
-directly into the store. This state can then be passed back to `react-map-gl`
-without any transformation.
+If you're using redux, it is very easy to hook this component up to store state in the redux state tree.
+The simplest way is to take all properties passed to the `onChangeViewport` function property and add them
```
Comment (abmai) [inline README.md]: Oops, `onChangeViewport` => `onViewportChange`

**Pull request merged**